pkg/client/client_venafi_cloud.go

```diff
@@ -0,0 +1,377 @@
+package client
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
```

Comment:
Nitpick: We should try and not use the io/ioutil, in new code since it's deprecated

Reply:
Good to just use `io.ReadAll`?